### DISCUSSION TITLE
Add multiple regex for allow_origin_regex

### DISCUSSION THREAD

**Discussion by odiseo0** [encode/starlette#1740] [Ideas] [answered]
I think some people may need this, even though I don't know if there is a lot of people with this use case

I implemented this multiple regex and currently it's being used in production but I'd like to have the feature natively in Starlette

**Comment by adriangb** [comment, marked as answer]
Can't you combine multiple regexes into a single regex?

**Reply by odiseo0** [reply]
I thought that didn't exist, sorry. I'll try that right now

**Reply by odiseo0** [reply]
It does work indeed, no need to do what I have done 🤡

**Reply by adriangb** [reply]
Mind marking this as the answer then?

**Reply by odiseo0** [reply]
Thanks
